core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/visualization/treemap/TreeMapOpDesc.scala

```diff
@@ -0,0 +1,103 @@
+package edu.uci.ics.amber.operator.visualization.treemap
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.amber.operator.PythonOperatorDescriptor
+import edu.uci.ics.amber.core.tuple.{AttributeType, Schema}
+import edu.uci.ics.amber.core.workflow.OutputPort.OutputMode
+import edu.uci.ics.amber.core.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.amber.operator.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.amber.operator.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.amber.operator.visualization.hierarchychart.HierarchySection
+
+class TreeMapOpDesc extends PythonOperatorDescriptor {
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Attribute")
+  @AutofillAttributeName
+  var textColumn: String = ""
+
+  var hierarchy: List[HierarchySection] = List()
+
+  override def getOutputSchemas(
+      inputSchemas: Map[PortIdentity, Schema]
+  ): Map[PortIdentity, Schema] = {
+    val outputSchema = Schema()
+      .add("html-content", AttributeType.STRING)
+    Map(operatorInfo.outputPorts.head.id -> outputSchema)
+    Map(operatorInfo.outputPorts.head.id -> outputSchema)
+  }
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Tree Map",
+      "Visualize Data into a Tree Map",
+      OperatorGroupConstants.VISUALIZATION_GROUP,
```

Comment:
There are subgroups in the visualization group. Currently your Tree Map operator stands out without a group (see the screenshot below).
<img width="291" alt="image" src="https://github.com/user-attachments/assets/23eb4c9f-2465-455f-883b-5b5338e782d9" />